app/views/expenses/index.html.erb

```diff
@@ -19,12 +19,15 @@
         <td><%= exp.price %></td>
         <td><%= exp.paid_on %></td>
         <td>
-          <%= link_to t('expenses.index.edit'), edit_expense_path(exp.id), class: "btn btn-outline-primary"%>
-          <%= link_to t('expenses.index.destroy'), expense_path(exp.id), method: :delete,
-                        data: { confirm: I18n.t('expenses.delete.confirmation') }, class: "btn btn-outline-danger"%>
+          <%= link_to t('expenses.index.edit'), edit_expense_path(exp.id), class: "btn btn-outline-primary" %>
+          <button type="button" class="btn btn-outline-danger" data-bs-toggle="modal" data-bs-target="#exampleModal<%= exp.id %>">
+            <%= t('expenses.index.destroy') %>
+          </button>
         </td>
+        <%= render partial: 'modal', locals: { idx: exp } %>
```

Comment:
Ah, it comes from here. So maybe some more descriptive name would be in favour.

Reply:
I used this because on first i use `expense` and it's very misleading. I decided to use idx, but you are right it can be better. :)